modules/serverless-applications-checking-ingress-istio-routing.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * knative-serving/debugging-serverless-applications.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-applications-checking-ingress-istio-routing_{context}"]
+= Checking Ingress and Istio routing
+
+Sometimes, when Istio is used as an Ingress layer, the Ingress and Istio routing have issues. You can see the details on them using the `oc` tool.
+
+.Procedure
+
+. List all Ingress resources and their corresponding labels:
++
+[source,terminal]
+----
+$ oc get ingresses.networking.internal.knative.dev -o=custom-columns='NAME:.metadata.name,LABELS:.metadata.labels'
+----
++
+.Example output
+[source,terminal]
+----
+NAME            LABELS
+helloworld-go   map[serving.knative.dev/route:helloworld-go serving.knative.dev/routeNamespace:default serving.knative.dev/service:helloworld-go]
+----
++
+In this output, labels `serving.knative.dev/route` and `serving.knative.dev/routeNamespace` indicate the `Route` where the Ingress resource resides. Your `Route` and Ingress should be listed.
++
+If your Ingress does not exist, the route controller assumes that the `Revision` objects targeted by your `Route` or `Service` are not ready. Proceed to the following procedures sections to diagnose `Revision` readiness status.
```

Comment:
"Proceed to the following procedures sections"

Considering modularized content and not knowing for sure where the module might appear in the future, Is there a way to say this without using a directional term? Maybe "Proceed with other debugging procedures" or similar?